test/bundler/package.json

```diff
@@ -1,10 +1,21 @@
 {
   "private": true,
   "scripts": {
-    "build": "npm run build.component-lib",
+    "build": "npm run build.component-lib && npm run build.vite",
     "build.component-lib": "cd component-library && npm ci && npm run build && cd ..",
-    "clean": "npm run clean.component-lib",
+    "build.vite": "cd vite-bundle-test && npm ci && npm run build && cd ..",
+    "clean": "npm run clean.component-lib && npm run clean.vite",
     "clean.component-lib": "cd component-library && npm run clean && cd ../",
-    "start": "npm run clean && npm run build"
+    "clean.vite": "cd vite-bundle-test && npm run clean && cd ../",
+    "karma": "karma start karma.config.ts",
+    "start": "npm run clean && npm run build && npm run karma"
```

Comment:
Whoops - forgot to add `npm ci` here. I'll take care of that on Tues and fix the downstreams